### PR TITLE
Projects To Footer HTML/CSS

### PR DESCRIPTION
**Pull request by Kccendana**
- add HTML and CSS for the project, footer, and get-started section
- used grid and flexbox for the layout
- change div to section tag in works class and about-section
- use Grid in work class
- use flexbox in the projects class
- change p tag to h3 tag in project name
- change div to button in 'see this project' and 'get cv' and added hover and active effect
- aligns items using flexbox in about-section

### CODE REVIEW

**Review by shella12 — COMMENTED** [review]
Hi @Kccendana , 

#  Status: Invalidated :red_circle: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] has a live demo link but the project is not deployed. Please delete the live demo section if you have decided not to deploy the application. If project requirements say you should deploy, then you should put the link.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -19,22 +19,227 @@
...
+                    <p class="t-description">
+                        If you like what you see and have a project you need coded, don't hesitate to contact me.
+                    </p>
+                    <div class="strt-collab-btn button">Start collaboration</div>
+                </div>
+                
+                <div class="intro-accent"><img src="images/image_geometry_2.svg" alt="geometric image"></div>
+            </section>
+            <div class="works">
+                <h2 class="title m8-4">Projects</h2>
```
Comment (abdulhamiid) [inline index.html]: Kindly make the container of the works section a `section` tag
* It represents a generic standalone section of the document, Let's use it 🤗. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. 💪

Also, according to the project requirement, we should use CSS `display:grid` in the `Works` section to lay out its content. please kindly implement that. 🛠

Requirement screenshot 👇 

image

File: index.html [hunk 2]
```diff
@@ -19,22 +19,227 @@
...
+                                <li>Ruby on Rails</li>
+                                <li>JavaScript</li>
+                            </ul>
+                        </div>
+                        <div class="see-proj">
+                            <p>See this project</p>
+                            <img src="images/ic_arrow_right.svg" alt="right arrow image">
+                        </div>
+                    </div>
+                </section>
```
Comment (abdulhamiid) [inline index.html]: According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), it is important that we use CSS `Flexbox`  to lay out the content of the individual cards.

Please kindly use Flexbox on the containers => `<section class="projects">` 🛠👍

File: index.html [hunk 3]
```diff
@@ -19,22 +19,227 @@
...
+            </section>
+            <div class="works">
+                <h2 class="title m8-4">Projects</h2>
+                <section class="projects">
+                    <div class="project-img">
+                        <img src="images/Rectangle 21.png" alt="upview computer">
+                    </div>
+                    <div class="project-name">
+                        <p>Project name goes here</p>
+                        <div class="tech-stack">
```
Comment (abdulhamiid) [inline index.html]: Please, kindly use a header tag `hx` for all the project titles i.e `Project name goes here`
* This way will be semantically correct and the screen readers and other technologies will thank you. 😎

File: index.html [hunk 4]
```diff
@@ -19,22 +19,227 @@
...
+                            <ul>
+                                <li>HTML/CSS</li>
+                                <li>Ruby on Rails</li>
+                                <li>JavaScript</li>
+                            </ul>
+                        </div>
+                        <div class="see-proj">
+                            <p>See this project</p>
+                            <img src="images/ic_arrow_right.svg" alt="right arrow image">
```
Comment (abdulhamiid) [inline index.html]: Kindly make the button (e.g., "See project") a link `a` or `button` tag. this way will be semantically correct, and let's follow the button styling: Active, Hover, and Pressed (see design)

image

File: index.html [hunk 5]
```diff
@@ -19,22 +19,227 @@
...
+                        </div>
+                    </div>
+                </section>
+
             </div>
-            
-            <div class="intro-accent"><img src="images/image_geometry_2.svg" alt=""></div>
-        </section>
+            <div class="about-section pad20-16">
+                <h2 class="title">About</h2>
```
Comment (abdulhamiid) [inline index.html]: Kindly make the container of the `About` section a `section` tag, This way will be semantically correct. 🛠🤗

* You can [check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.

Also, let's align the content of the `section` with Flexbox according to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) 👽

File: index.html [hunk 6]
```diff
@@ -19,22 +19,227 @@
...
+                <p class="t-description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                <div class="button d-flex get-cv">
+                    <span>Get my CV</span>
+                    <img src="images/ic_download.svg" alt="download image">
+                </div>
```
Comment (abdulhamiid) [inline index.html]: Kindly make the button "Get my CV" a link `a` or `button` tag for semantic purposes, and let's follow the button styling: Active, Hover, and Pressed. 🛠👍

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @Kccendana 👋, 
Alt Text
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

[REQUIRED] -  I noticed that the links you included for the `Live Demo` does not navigate to your portfolio project.

image

- [ ] As you may know, links are an essential part of any project, and it's crucial to ensure that they are functional and lead to the intended content. In this case, the broken link may hinder the ability of others to access important information related to your project. 

- [ ] Since your project was not deployed, kindly consider removing the `Live Demo Link` to make your readme professional. 

- [ ] Or you can simply put `Live Demo is not available`  instead🤗👍 

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @Kccendana, 




Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡
 
✅ Descriptive PR 👍 
✅ Linters are passing 👏 
✅ Nice design :heavy_check_mark: 

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**